com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -478,14 +480,18 @@ public void SpawnAsPlayerObject(ulong clientId, bool destroyWithScene = false)
         }
 
         /// <summary>
-        /// Despawns this GameObject and destroys it for other clients. This should be used if the object should be kept on the server
+        /// Despawns the parent <see cref="GameObject"/> of this <see cref="NetworkObject"/> and sends a destroy message for it to all connected clients.
```

Comment:
```suggestion
        /// Despawns the <see cref="GameObject"/> of this <see cref="NetworkObject"/> and sends a destroy message for it to all connected clients.
```
I don't think we should use parent here. Parent in Unity means a hierarchical parent in the GameObject hierarchy. A novice Unity developer could misinterpret this as that the parent GameObject of the GameObject with the NetworkObject gets despawned which is not true.